### DISCUSSION TITLE
0.4.1

### DISCUSSION THREAD

**Discussion by he3als** [Atlas-OS/Atlas#1246] [Announcements]
# Atlas v0.4.1

Version v0.4.1 brings more customizability, stability, and compatibility to Atlas, with many critical bug fixes and more Windows feature support. Most notably, [Windows 11 24H2](https://support.microsoft.com/topic/windows-11-version-24h2-update-history-0929c747-1815-4543-8461-0160d16f15e5) support is now included, and Edge removal is fixed. 😊

## 📦 Upgrades

> [!IMPORTANT]  
> **Due to upgrade support still being a work in progress, this version of Atlas requires a complete Windows reinstall to upgrade from older versions**.

If you're currently using v0.4.0, you can upgrade to v0.5.0 once it's released, which will include all the new features and fixes from this release. Additionally, if you're on version 0.4.0 23H2, you can update to 24H2 by upgrading to version 0.5.0 first when it becomes available. Remember that there's no set date for the release of version 0.5.0 yet.

This means that v0.4.0 users will have to wait until the v0.5.0 release to upgrade. See the ["Upgrading Atlas"](https://docs.atlasos.net/general-faq/updating-atlas/) page for more information.

## ⬇️ Installation

See the ["Installing AtlasOS"](https://docs.atlasos.net/getting-started/installation/) guide on how to install.

### Changes & Features
- Add Windows 11 24H2 support (23H2 is now unsupported)
- Keep SysMain enabled by default for responsiveness
- Disable the UCPD driver
- Make scripts that don't need TrustedInstaller not use it
- Add Copilot PWA support to Copilot scripts
- Use NanaZip instead of 7-Zip
- Desktop folder reorganization
   - Move scripts that change services into the 'Advanced Configuration' folder
   - Un-nest folders that were previously inside other folders in 'General Configuration'
- Add 'Snap Layouts' Registry files to optionally disable them
- Add PowerShell 7 and UniGetUI into the 'Install Software' script
- Add Google Chrome ARM64 support
- Add proper AME Wizard v0.7.5+ support
- Clean component store after installing default optional features to reduce disk usage
- Remove problematic service scripts
- Ability to toggle the 'Give access to' context menu when enabling file sharing
- Disable 'Dynamic Lighting' by default so peripheral onboard profiles aren't changed
- Improve search indexing
   - Respect power modes when search indexing
   - Enable by default, except with a very minimal configuration, for compatibility
   - Ability to search for Atlas desktop files in the Windows Search
- Make Fax & Scan optional as it takes a long time to install when enabling printing
- Add scripts to enable location services, including Find My Device
- Add troubleshooting script to fix Windows components (using DISM & SFC)
- Add further policies to disable data collection
- Disable Recall on supported devices by default
- Disable automatic Store app archival by default
- Disable PATH length limit
- Ability to toggle the app icon on thumbnails
- Add toggle for 'Extract All'
- Rewrite many PowerShell scripts, and rewrite many batch scripts into PowerShell
- Make applying the theme faster and more reliable

### Bug Fixes
- Fix the deny option not working when prompted to install Edge for some scripts
- Fix Edge removal no longer working (update EdgeRemover to v1.9.3)
- Fix new users not having optimisations applied
- Fix enabling printing & notifications not unhiding Settings pages
- Re-enable Bluetooth advertising, which isn't *actual advertising*
- Fix DISM 'Sources' error with RestoreHealth
- Allow users to configure AutoPlay options
- Fix slideshow lock-screen issues
- Don't remove Windows Media Player Legacy by default for compatibility
- Remove legacy WMIC usage for compatibility
- Fix touchscreen gestures by default (enable edge swipe)
- Allow users to change USB settings in Windows Settings
- Fix false antivirus detections on some default shortcuts in the Atlas folder
- Fix the 'Accounts' Settings page disappearing after updates
- Fix PcaSvc and legacy speech recognition support
- Fix 'Receive updates for other Microsoft apps' toggle being not changeable
- Fix errors when applying service backup Registry files
- Support for multiple CPUs (sockets) in some scripts

...and lots more minor fixes and improvements!
You can see the [complete commit log](https://github.com/Atlas-OS/Atlas/compare/0.4.0...0.4.1).

## ℹ️ Info
🌐 **Website**: https://atlasos.net/
💬 **Forums**: https://forum.atlasos.net/
💬 **Discord**: https://discord.atlasos.net/
📚 **Docs**: https://docs.atlasos.net
☕ **Ko-Fi (donations):** https://ko-fi.com/atlasos

<hr /><em>This discussion was created from the release <a href='https://github.com/Atlas-OS/Atlas/releases/tag/0.4.1'>0.4.1</a>.</em>

**Comment by lovi-dzen** [comment]
Thanks for the update, I want to put it on a new system, can you please tell me if it is tested on Windows LTSC 24H2 (Build 26100.1742)? Can I put Atlas on it? Or is it better to use Winodws Pro 24H2 (Build 26100.2033) ? I want to install on LTSC, but if you say no, I won't)

**Reply by Ast3risk-ops** [reply]
LTSC will never be supported.

**Reply by lovi-dzen** [reply]
> LTSC will never be supported.

Why? Can you tell me what that has to do with?

**Reply by Ast3risk-ops** [reply]
> > LTSC will never be supported.
> 
> Why? Can you tell me what that has to do with?

There's more technical reasons I don't know, but LTSC is stripped down already, why would you want Atlas?

**Reply by lovi-dzen** [reply]
Because Atlas is not just about cutting components out of the system, it's about optimization, speed and tweaks. I have an old system, so I want LTSC to be fast too

**Reply by trolljoe** [reply]
> Thanks for the update, I want to put it on a new system, can you please tell me if it is tested on Windows LTSC 24H2 (Build 26100.1742)? Can I put Atlas on it? Or is it better to use Winodws Pro 24H2 (Build 26100.2033) ? I want to install on LTSC, but if you say no, I won't)

if you want a technical answer why it won't get supported, then there's a few reasons:

1. to actually activate/get the LTSC iso, its locked behind a subscription service, and the only way to get it for free is through piracy websites.

2. it's hard to maintain several different windows versions, we already have to maintain windows 10 and windows 11, adding ltsc just makes things harder.

**Reply by lovi-dzen** [reply]
Thanks for the explanations, I will use windows pro

**Reply by Aolzarin** [reply]
Will there be a new update soon? I am using v0.4.1. Is it normal that there is no update for a long time?

**Comment by donglihui** [comment]
![PixPin_2024-10-10_09-07-06](https://github.com/user-attachments/assets/6de41cd4-8f44-4995-87b2-13776da6331f)
There seems to be something wrong with this playbook.

**Reply by Ast3risk-ops** [reply]
I don't speak Chinese. Don't assume people on the team can speak it. What does this say in English?

**Reply by kamild1996** [reply]
The "FreshInstall" part sounds familiar. Make sure you re-downloaded AME Wizard from the Atlas's website, you probably have an old version.

**Reply by MengYuDream** [reply]
> The "FreshInstall" part sounds familiar. Make sure you re-downloaded AME Wizard from the Atlas's website, you probably have an old version.

Solved it! Thank you.

**Reply by donglihui** [reply]
> The "FreshInstall" part sounds familiar. Make sure you re-downloaded AME Wizard from the Atlas's website, you probably have an old version.

yes，you are right

**Comment by candrapersada** [comment]
is it possible to update `Atlas v0.5` from `v0.4.0` for `Windows 10`?

**Reply by your-angel** [reply]
Bump

**Reply by H0ggu** [reply]
https://docs.atlasos.net/general-faq/updating-atlas/
The faq covers this

**Comment by wixaw** [comment]
Hello,
When will version 5 be available?
Thank you in advance.

**Reply by LumiFae** [reply]
Considering this got posted 2 weeks ago, be a bit more patient and wait for it to be publically mentioned.

**Comment by Evillord36** [comment]
5 desktop pc working on these atlast 0.4.1. it so nice and legit. hope to install the latest version

**Reply by your-angel** [reply]
Sounds good. Report any bugs if you encounter them.

**Comment by NoT-ManDone** [comment]
Atlas 5 when?

**Comment by Deathbedkizz** [comment]
When **atlas 0.5.0** is released,will i need to **reinstall** Windows in order to update?

**Reply by candrapersada** [reply]
for windows 10 or windows 11?

**Reply by Deathbedkizz** [reply]
win11

**Comment by Aolzarin** [comment]
Windows 11
________________________________
Gönderen: candrapersada ***@***.***>
Gönderildi: 25 Mayıs 2025 Pazar 09:05
Kime: Atlas-OS/Atlas ***@***.***>
Bilgi: Fatih ***@***.***>; Comment ***@***.***>
Konu: Re: [Atlas-OS/Atlas] 0.4.1 (Discussion #1246)


for windows 10 or windows 11?

**Comment by PlamenG0527** [comment]
How to fix this? 
![2025-05-31_222006](https://github.com/user-attachments/assets/07f329f8-484f-4a5e-94f5-53448d0f0e3f)

**Reply by Ast3risk-ops** [reply]
Create a new post instead of commenting on an existing one.

**Reply by PlamenG0527** [reply]
the chinese guy was asking the same as i did and is not a big deal if you have info why is this eror just to reply or we should create 1000 of new topics where can get one answer- this happen in version 24h2

**Reply by Ast3risk-ops** [reply]
> the chinese guy was asking the same as i did and is not a big deal if you have info why is this eror just to reply or we should create 1000 of new topics where can get one answer- this happen in version 24h2

Because nobody will help you if you comment here. This thread is not for issues. Make a dedicated thread if you actually want support.

**Comment by insignificantcharacter** [comment]
Windows 10 22h2 Best 19045.xxxx build version for atlas 0.41 pls?
